Add Html and css files

Lasten-Ernest opened the pull request:
This pull request is for Hello world Microverse project. 
I have added the linter configuration files for lighthouse, webhint and stylelint that are used for conducting various tests like indentation errors and responsiveness.
I also added the CSS styles for the html header. And then added the test.md file in .gitignore file to restrict it from getting pushed into GitHub repository.

Lasten-Ernest committed:
Add Html and css files

Lasten-Ernest committed:
Add linters file

Gambit142 commented on README.md:
```diff
@@ -1 +1,37 @@
+Hello world
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+
```
- [ ] (Optional) It would be nice to include a Setup section in your `README.md` file where you describe how to clone your project and run it locally. It should loo like this:

```
git clone (ssh/http link)
cd <Folder(name of repository)>
Open with Vscode if it is installed
```

Gambit142 reviewed (changes requested):
Hi @Lasten-Ernest 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- All linters checks passed ✔️
- Commit messages are meaningful ✔️
- Used Microverse `README.md` template ✔️

## Required Changes ♻️
- Kindly make the summary of your pull request more descriptive. It should describe the changes you introduced to the project. 👍

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- You can also consider the optional changes I made under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

Lasten-Ernest committed:
Configure linters

Lasten-Ernest commented:
Added linter configuration files so that I can run the tests for example the lighthouse.
Added the test.md to .gitignore file so that it should be omitted when pushing to repository branch. 
Added the the css for the h1 header.

Lasten-Ernest committed:
Update README.md

Add steps to follow in order to clone and use this project

Lasten-Ernest committed:
Update README.md

uwadonat reviewed (approved):
Hi **@[Lasten-Ernest](https://github.com/Lasten-Ernest),** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

Lasten-Ernest commented:
Thanks

On Sat, 23 Apr 2022, 12:09 pm Donat Uwamahoro, ***@***.***>
wrote:

> ***@***.**** approved this pull request.
>
> Hi ***@***.*** <https://github.com/Lasten-Ernest>,*
>
> Your project is complete! There is nothing else to say other than... it's
> time to merge it [image: :shipit:]
> Congratulations! 🎉
> Status: Approved ✔️✔️✔️
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Lasten-Ernest/Hello-World/pull/1#pullrequestreview-950950996>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AN6ZWYSI7H5US7JDEXH3Y7TVGPD4XANCNFSM5UCHYK4Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

omar-muhamad commented on README.md:
```diff
@@ -1 +1,40 @@
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+Hello world
+
```
- [ ] Please add The project name which is `Hello Microverse` right after the `#` instead of `# Project Name` to be `# Hello Microverse` because it will look more professional like that.
- [ ] Please add the project description directly below the project name to make your README file more professional.

```
# Hello Microvese

> This is my first project with Microverse, In this project, I learned how to install linters correctly, make an index page, and link a style CSS file to it
```

omar-muhamad commented on README.md:
```diff
@@ -1 +1,40 @@
+- HTML
+- CSS
+
+# To use this project, just clone it into your pc by using the following steps:
+- git clone (https://github.com/Lasten-Ernest/Hello-World/pull/1)
+- cd <Folder(Hello-World)
+- Open with Vscode if it is installed
```
- [ ] It would be much better professionally if you name this section `## Getting Started` and put the text line 13 as a normal text below this title without the `#` in the beginning.

omar-muhamad reviewed (changes requested):
# Changes Required ♻️
### Hi @Lasten-Ernest,

Nice job, you are almost there.

![](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please change the branch name to be a meaningful name with the name of the feature you did to the project, not the word feature or sub-feature like you did.
for example you can name it `setup-project`.
[Hint]: you can check this [article](https://devconnected.com/how-to-change-branch-name-on-git/) to help you.

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Lasten-Ernest commented:
Thanks for your response.

I did this Hello-Microverse project and it was approved but I forgot to
submit it as complete and since I am repeating the block due to my laptop
hardware failure which resulted in not finishing other parts of the first
block. But I was surprised to see that changing the repository branch name
is a requirement while it was not during the first review last week.

So I just have created a pull request 2 and modified the readMe file as
well.



On Wed, Apr 27, 2022 at 11:06 AM Omar Muhammad ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> Changes Required ♻️ Hi @Lasten-Ernest <https://github.com/Lasten-Ernest>,
>
> Nice job, you are almost there.
>
>
> <https://camo.githubusercontent.com/34482800e1b92cb79ddca06edb4051edc407cb190f896e1e949e596aa9073851/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966>
> (Highlights) Good Points: 👍
>
>    - You documented your work professionally. ✔
>    - Github flow is used correctly. ✔
>    - All linters checks are ok. ✔
>
> (Changes Required) Aspects to improve: ♻️
>
>    -
>
>    Please change the branch name to be a meaningful name with the name of
>    the feature you did to the project, not the word feature or sub-feature
>    like you did.
>    for example you can name it setup-project.
>    [Hint]: you can check this article
>    <https://devconnected.com/how-to-change-branch-name-on-git/> to help
>    you.
>    -
>
>    Please check the inline comments below.
>
> [Optional] suggestions:
>
>    - Nothing to mention.
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> *Cheers and Happy coding!👏👏👏*
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear, and please remember to tag me in your question so I can
> receive the notification**
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have
> a limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.
> ------------------------------
>
> In README.md
> <https://github.com/Lasten-Ernest/Hello-World/pull/1#discussion_r859548230>
> :
>
> > +# Project Name
>
> +
>
> +Hello world
>
> +
>
>
>
>    - Please add The project name which is Hello Microverse right after
>    the # instead of # Project Name to be # Hello Microverse because it
>    will look more professional like that.
>    - Please add the project description directly below the project name
>    to make your README file more professional.
>
> # Hello Microvese
>
>
>
> > This is my first project with Microverse, In this project, I learned how to install linters correctly, make an index page, and link a style CSS file to it
>
>
> ------------------------------
>
> In README.md
> <https://github.com/Lasten-Ernest/Hello-World/pull/1#discussion_r859551447>
> :
>
> > +# To use this project, just clone it into your pc by using the following steps:
>
> +- git clone (https://github.com/Lasten-Ernest/Hello-World/pull/1)
>
> +- cd <Folder(Hello-World)
>
> +- Open with Vscode if it is installed
>
>
>
>    - It would be much better professionally if you name this section ##
>    Getting Started and put the text line 13 as a normal text below this
>    title without the # in the beginning.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Lasten-Ernest/Hello-World/pull/1#pullrequestreview-954546829>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AN6ZWYTTMAM74FE6IWVBVEDVHD7STANCNFSM5UCHYK4Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
